src/lib/check-flags.js

```diff
@@ -0,0 +1,13 @@
+export default (item, newItem) => {
+    // check flags
+    const itemFlags = item.flags ? item.flags : false
+    const newItemFlags = newItem.flags ? newItem.flags : false
+    if (!itemFlags && !newItemFlags) {
+      return true
+    } else if (!itemFlags && newItemFlags || itemFlags && !newItemFlags) {

```

Comment:
esses AND e OR aqui estão ambíguos, teria que ter um parênteses na segunda parte

Reply:
E o StandardJS te mostraria isso antes de mim inclusive..

Reply:
Ue, eu estou com isso instalado no meu vs code

Reply:
Tem que instalar as dependências (sempre) do repositório também
Fez isso?